src/styles/blocks/card.scss

```diff
@@ -0,0 +1,84 @@
+$grey-color: #616070;
+$blue-color: #00acdc;
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 16px;
+  padding-top: 32px;
+
+  &--width {
+    width: 200px;
+  }
+
+  &__photo {
+    display: block;
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__header {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin: 0 0 16px;
+    color: $grey-color;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    margin-bottom: 16px;
+
+  }
+
+  &__title {
+    color: $grey-color;
+    font-size: 12px;
+    font-weight: 400;
+
+  }
+
+  &__sum {
+    font-size: 16px;
+    font-weight: 700;
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 12px 70px;
+    border: 1px solid #f3f3f3;
```

Comment:
Creating a border for an inactive button wasn't the task requirement. If it is to be created, I think it should be the same color as the button itself (or background) when we do not hover over it.